client/src/components/Panels/Common/ToolPanelLabel.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <div
+        class="tool-panel-label"
+        :title="description"
+        v-b-tooltip.topright.hover
+        @mouseover="hover = true"
+        @mouseleave="hover = false"
+    >
+        {{ definition.text }}
+        <ToolPanelLinks :show="hover" :links="definition.links" />
```

Comment:
Do you need a `v-if="definition.links"` here ?

Reply:
I have the v-if on the inside of the component - I feel like that is a bit more consistent but might be bad form?

Reply:
Yeah, that seems better, although I would have assumed that links being null would violate the prop type being `Object` ... but I see no warning so that just confirms I don't understand vue 😆

Reply:
@mvdbeek In js, `null` *is* an object!  :)

Reply:
Was about to ask that ...